### PR TITLE
PD-272 stork app warnings

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md [hunk 1]
```diff
@@ -317,7 +321,14 @@ mysql-6d69b99774-2bv4m   0/1     Pending   0          3s
...
 ## Clone an Application
 
-You can clone an application to a different namespace, or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+You can clone an application to a different namespace or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+
+{{<info>}}
+**NOTE:**
+
+* If you're cloning an application across namespaces on OpenShift, you must create both namespaces with the same `fs group id` annotations.
+* Distributed apps, such as Cassandra, may use the same identity on the destination, causing disruption.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md]: @disrani-px This wasn't very clear to me. destination namespace? What kind of disruption?

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md]: Yes, destination namespace. In cassandra, the nodes end up joining the source cluster with the same node ids and the source cluster nodes get confused

File: content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md [hunk 2]
```diff
@@ -317,7 +321,14 @@ mysql-6d69b99774-2bv4m   0/1     Pending   0          3s
 
 ## Clone an Application
 
-You can clone an application to a different namespace, or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+You can clone an application to a different namespace or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+
+{{<info>}}
+**NOTE:**
+
+* If you're cloning an application across namespaces on OpenShift, you must create both namespaces with the same `fs group id` annotations.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md]: @disrani-px let's go over this, I don't understand the context of this annotation:

> Example of annotations that need to be the same in the namespaces:
> 
> openshift.io/sa.scc.supplemental-groups: 1001990000/10000
>     openshift.io/sa.scc.uid-range: 1001990000/10000
>

File: content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md [hunk 3]
```diff
@@ -317,7 +328,21 @@ mysql-6d69b99774-2bv4m   0/1     Pending   0          3s
 
 ## Clone an Application
 
-You can clone an application to a different namespace, or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+You can clone an application to a different namespace or within the same namespace. You must create the `ApplicationClone` object in the **admin** namespace, which is `kube-system` by default.
+
+{{<info>}}
+**NOTE:**
+
+* If you're restoring an application across namespaces on OpenShift, you must modify your destination namespace to include the same supplemental group annotation values as your source namespace:
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/stateful-applications/_index.md]: `If you are cloning...`

**Pull request merged**